ui/v2.5/src/components/Galleries/GalleryCard.tsx

```diff
@@ -14,6 +14,45 @@ import { faBox, faPlayCircle, faTag } from "@fortawesome/free-solid-svg-icons";
 import { galleryTitle } from "src/core/galleries";
 import ScreenUtils from "src/utils/screen";
 import { StudioOverlay } from "../Shared/GridCard/StudioOverlay";
+import { GalleryPreviewScrubber } from "./GalleryPreviewScrubber";
+import cx from "classnames";
+import { useHistory } from "react-router-dom";
+
+interface IScenePreviewProps {
+  isPortrait?: boolean;
+  gallery: GQL.SlimGalleryDataFragment;
+  onScrubberClick?: (index: number) => void;
+}
+
+export const GalleryPreview: React.FC<IScenePreviewProps> = ({
+  gallery,
+  isPortrait = false,
+  onScrubberClick,
+}) => {
+  const [imgSrc, setImgSrc] = useState<string | undefined>(
+    gallery.cover?.paths.thumbnail ?? undefined
+  );
+
+  return (
+    <div className={cx("gallery-card-cover", { portrait: isPortrait })}>
```

Comment:
I noticed the height and width dimensions were removed from the cover property, and it currently looks like `isPortrait` is always set to false. How did you plan to identify when a gallery card has a portrait image?

Reply:
Looks like it was a vestigial artifact from the `ScenePreview` that it was copied from. As far as I can determine, there's no need for it in the Gallery preview/scrubber, and the styling needs to accommodate for both orientations anyway.